stepconf/stepconf.go

```diff
@@ -160,6 +172,15 @@ func setField(field reflect.Value, value, constraint string) error {
 		field.SetFloat(f)
 	case reflect.Slice:
 		field.Set(reflect.ValueOf(strings.Split(value, "|")))
+	case reflect.Ptr:
+		// if the field is a ptr type then we need to initialize a pointer address first
+		// and then get the type the address points to and set a new non-nil zero type to it
```

Comment:
Did you mean: If field is a pointer type, then set its value to be a pointer to a new zero value, matching field underlying type.